data/json/items/tool/workshop.json

```diff
@@ -452,6 +452,21 @@
     "weapon_category": [ "BATONS" ],
     "melee_damage": { "bash": 9 }
   },
+  {
+    "id": "hammer_crude",
+    "type": "TOOL",
+    "name": { "str": "crude hammer" },
+    "description": "A simple crude steel hammer.  It is not as sophisticated or useful as a properly made alternative but it does get the job of hammering in nails well enough.",
```

Comment:
The description is very illustrative.  Hammering in nails,  fine.  Forging,  no.

Reply:
I'm find changing the description. The thing is, creating a smithing hammer is more about metal quality than anything else. They actually aren't hard to make, they tend to be blocky and require only a little shaping on either end to create a flat and rounded surface respectively. Making a claw hammer would be far more difficult.

Reply:
Fixed the description and increased the weight a bit to represent a more conventional smithing hammer.